cypress/support/commands/action.js

```diff
@@ -82,11 +82,12 @@ Cypress.Commands.add('executeQuickAction', (actionName, modal = false, isDialogE
   cy.waitForSaveIndicator();
 });
 
-Cypress.Commands.add('openPickingHUSelectionWindow', function() {
+Cypress.Commands.add('executeQuickActionWithRightSideTable', actionName => {
+  cy.log(`executeQuickActionWithRightSideTable ${actionName}`);
   const layoutAlias = 'layout_' + new Date().getTime();
```

Comment:
new Date().getTime() can we extract it? as I have seen it used in several places.

Reply:
dunno is this should be extracted. it should only be used inside cypress functions when awaiting the response to a request, so it doesn't bother me much.